.env.development

```diff
@@ -0,0 +1,17 @@
+NODE_ENV='development'
+PORT=2002
```

Comment:
@stvstnfrd  I picked this port as learning is 2000, and authoring is 2001 so I just picked the next one.